gamemodes/terrortown/gamemode/init.lua

```diff
@@ -127,6 +130,13 @@ CreateConVar("ttt_credits_award_size", "1")
 CreateConVar("ttt_credits_award_repeat", "1")
 CreateConVar("ttt_credits_detectivekill", "1")
 
+-- Cursed credits --
+CreateConVar("ttt_cur_credits_starting", "1")
```

Comment:
Should they start with 0? The other specials do (other than Detraitor)

Reply:
I don't think so considering they don't really have access to anything that helps them win while alive like the other roles